src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -171,6 +171,17 @@ void NotationViewInputController::initZoom()
     currentNotation()->viewState()->setMatrixInited(true);
 }
 
+void NotationViewInputController::initCanvasPos()
+{
+    double canvasMargin = 10 * DPMM; // 1 centimetre
```

Comment:
I assume it's intentional that this cm is relative to the score size, and not always corresponds to 1 cm on the screen (for example when the default zoom percentage is not 100%).

Maybe it's a bit much though. I would personally prefer something closer to `MScore::horizontalPageGapOdd`.